benchmark/vochain_benchmark_test.go

```diff
@@ -27,214 +27,133 @@ const (
 	hexProcessID   = "0xe9d5e8d791f51179e218c606f83f5967ab272292a6dbda887853d81f7a1d5105"
 )
 
-func BenchmarkVochain(b *testing.B) {
+var zeroReq = &api.APIrequest{}
+
+func reset(r *api.APIrequest) {
```

Comment:
What do you think about defining this in api/types.go as a helper in the following way?

```golang
func (r *APIrequest) reset() {
	r = &APIrequest{}
}
```